servers/visual/visual_server_scene.cpp

```diff
@@ -1522,6 +1522,13 @@ void VisualServerScene::instance_geometry_set_material_override(RID p_instance,
 }
 
 void VisualServerScene::instance_geometry_set_draw_range(RID p_instance, float p_min, float p_max, float p_min_margin, float p_max_margin) {
+	Instance *instance = instance_owner.get(p_instance);
+	ERR_FAIL_COND(!instance);
+
+	instance->lod_begin = p_min;
+	instance->lod_end = p_max;
+	instance->lod_begin_hysteresis = p_min_margin;
+	instance->lod_end_hysteresis = p_max_margin;
 }
 void VisualServerScene::instance_geometry_set_as_instance_lod(RID p_instance, RID p_as_lod_of_instance) {
```

Comment:
This one should be removed, nothing ever calls it.

Reply:
That method is exposed to the scripting API – wouldn't removing it technically break compatibility?